tests/nlp-integration.sh

```diff
@@ -20,19 +20,7 @@ export SECRET_KEY
 
 export THINGENGINE_USE_TOKENIZER=local
 
-NLP_PORT=${NLP_PORT:-8400}
-TRAINING_PORT=${TRAINING_PORT:-8090}
 cat > $srcdir/secret_config.js <<EOF
```

Comment:
do you mean to remove the cat statement as well ?

Reply:
Yeah